relay-server/src/metric_stats.rs

```diff
@@ -201,11 +206,13 @@ impl MetricStats {
     }
 }
 
-#[cfg(all(test, feature = "processing"))]
```

Comment:
We want to enable most of the tests in any case, but right now we track cardinality only if processing is enabled so those tests are still kept only for processing, to avoid unused variables warnings.